src/compo/viirs_aod2ioda.py

```diff
@@ -311,7 +310,10 @@ def read(self):
                 self.qcfs = self.qcfs[mask_thin]
                 self.obs_time = self.obs_time[mask_thin]
 
-            #  Write out data
+            # create the final mask for output based on time window
```

Comment:
```suggestion
            # after the thinning above apply a mask based on time window
```

@mer-a-o is this the line you were referring to and did I interpret correctly please correct as needed @weiwilliam nothing else is needed correct

Reply:
i am also very confused how I can't add a suggestion and commit it